src/app/src/hooks/useTelemetry.tsx

```diff
@@ -0,0 +1,31 @@
+import { useCallback } from 'react';
+import { callApi } from '@app/utils/api';
+import type { EventProperties, TelemetryEventTypes } from '@promptfoo/telemetry';
+
+export function useTelemetry() {
+  const recordEvent = useCallback(
+    async (eventName: TelemetryEventTypes, properties: EventProperties) => {
+      if (!['0', 'false'].includes(import.meta.env.VITE_PROMPTFOO_DISABLE_TELEMETRY)) {
```

Comment:
Is this additional envar necessary?  The server side already won't record telemetry if `PROMPTFOO_DSIABLE_TELEMETRY` is set, so this new envar is just cosmetic.

Reply:
Good question. Not necessary but this disables the xhr from appearing in the network tab. I thought that people might be confused / unhappy if they set `PROMPTFOO_DISABLE_TELEMETRY` but still see a post to `/api/telemetry` with the response `{ success: true }`